SocketRocket/SRWebSocket.m

```diff
@@ -151,6 +153,8 @@ @implementation SRWebSocket {
     SRProxyConnect *_proxyConnect;
 }
 
+@synthesize readyState = _readyState;
```

Comment:
Curious, what does this `@synthesize` declaration do for this PR?

Reply:
can you remove atomic ? when i use infer scan my project, got them:


> Direct access to ivar `_readyState` of an atomic property at line 300, column 13. Accessing an ivar of an atomic property makes the property nonatomic.